apps/worker/src/queue.ts

```
@@ -0,0 +1,367 @@
+  depth: number = 0
+): Promise<string> {
+  const indent = '  '.repeat(depth);
+  let result = '';
+
+  switch (block.type) {
+    case 'paragraph':
+      result = (await convertInlineContent(block.content || [])) + '\n\n';
+      break;
+
+    case 'heading':
+      const level = block.attrs?.level || 1;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)